2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -352,7 +352,8 @@ $bold("Scenario: ")$
 )
 == Creazione scaffale
 $bold("Descrizione: ")$
-uno scaffale viene creato in base ai valori dati dall'utente e aggiunto nell'ambiente in una posizione valida specificata. Seccessivamente vengono creati i bin contenuti dallo scaffale e posizionati in esso.
+uno scaffale viene creato in base ai valori dati dall'utente quali: lunghezza, larghezza, profondità, numero di piani, colonne in cui è suddiviso e orientamento nel piano (orizzontale o verticale).
```

Comment:
"in base ai valori inseriti dall'utente" al posto di "in base ai valori dati dall'utente"